### PR TITLE
Updates to retry definition

### PR DESCRIPTION
**Pull request by jorgenj**
**Many thanks for submitting your Pull Request :heart:!**

**Please specify parts this PR updates:**

- [ x] Specification
- [ x] Schema
- [ x] Examples
- [ ] Extensions
- [ ] Roadmap
- [ ] Use Cases
- [ ] Community
- [ ] TCK
- [ ] Other

**What this PR does / why we need it**:

Address requested changes in #137 & #136

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

File: schema/retries.json [hunk 1]
```diff
@@ -28,9 +28,18 @@
...
+          "description": "Maximum time delay between retry attempts (ISO 8601 duration format)"
+        },
+        "increment": {
           "type": "string",
-          "description": "Multiplier value by which interval increases during each attempt (ISO 8601 time format)"
+          "description": "Static value by which the delay increases during each attempt (ISO 8601 time format)"
+        },
+        "multiplier": {
+          "type": ["number","string"],
+          "minimum": 0,
```
Comment (tsurdilo) [inline schema/retries.json:41]: add 
"minLength": 1 
since it can also be string type

File: schema/retries.json [hunk 2]
```diff
@@ -28,9 +28,18 @@
...
+          "description": "Float value, if specified the delay between retries is multiplied by this value."
```
Comment (tsurdilo) [inline schema/retries.json:42]: I would also define the decimal point precision if you require this to be a float. 
You can use "multipleOf" param when its number type.  Prob enough 1 decimal points in which case add
"multipleOf": 0.1
to multiplier

File: specification.md [hunk 3]
```diff
@@ -1349,8 +1349,10 @@ For more information, see the [Workflow Error Handling](#Workflow-Error-Handling
 | --- | --- | --- | --- |
 | name | Unique retry strategy name | string | yes |
 | delay | Time delay between retry attempts (ISO 8601 duration format) | string | no |
-| multiplier | Multiplier value by which interval increases during each attempt (ISO 8601 time format). For example: "PT3S" meaning the second attempt interval is increased by 3 seconds, the third interval by 6 seconds and so on | string | no |
 | maxAttempts | Maximum number of retry attempts. Value of 0 means no retries are performed | string or number | no |
+| maxDelay | Maximum amount of delay between retry attempts (ISO 8601 duration format) | string | no |
+| increment | Static duration which will be added to the delay between successive retries (ISO 8601 duration format) | string | no |
+| multiplier | Float value by which the delay is multiplied before each attempt. For example: "1.2" meaning that each successive delay is 20% longer than the previous delay.  For example, if delay is 'PT10S', then the delay between the first and second attempts will be 10 seconds, and the delay before the third attempt will be 12 seconds. | float or string | no |
```
Comment (tsurdilo) [inline specification.md:1355]: Add the decimal precision info here too (1 decimal point if you define it as such with  multipleOf)

File: specification.md [hunk 4]
```diff
@@ -1431,6 +1446,28 @@ Alternatively, `jitter` may be defined as an absolute value specified as an ISO
 8601 duration. This way, the maximum amount of random time added is fixed and
 will not increase as new attempts are made.
 
+The `maxDelay` property determines the maximum amount of delay that is desired between retry attempts, and is applied 
+after `increment`, `multiplier`, and `jitter` are applied.
```
Comment (tsurdilo) [inline specification.md:1450]: no need for the last "are applied. just end sentence after 'jitter'

**Review by tsurdilo — CHANGES_REQUESTED** [review]

**Comment by tsurdilo** [comment]
@jorgenj thanks for pr!

**Comment by tsurdilo** [comment]
@jorgenj please also add a row to the roadmap/README.md file that points out this update. Thanks.

**Review by ricardozanini — COMMENTED** [review]
LGTM as soon as @tsurdilo's comments are taken care of.

**Review by tsurdilo — APPROVED** [review]

**Comment by tsurdilo** [comment]
@jorgenj thanks for making the updates! It seems the DCO check is failing. Commit needs to be signed.

**Pull request merged**